autoware_utils_tf/test/cases/self_pose_listener.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright 2025 The Autoware Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware_utils_tf/self_pose_listener.hpp"
+
+#include <gtest/gtest.h>
+
+TEST(TestSelfPoseListener, Main)
+{
+}
```

Comment:
The test is empty. That’s because we can’t test it without writing an integration test. We would like to add the TODO comment at least.